dev/release/verify-release-candidate.sh

```diff
@@ -1198,5 +1198,5 @@ test_binary_distribution
 
 TEST_SUCCESS=yes
 
-echo 'Release candidate looks good!'
+echo "Release candidate number: ${RC_NUMBER} for version: ${VERSION} looks good!"
```

Comment:
It's not a strong opinion but how about simplifying it?

```suggestion
echo "Release candidate ${VERSION}-RC${RC_NUMBER} looks good!"
```